.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Fetch Git history
+        run: git fetch --no-tags --depth=50 origin main
+
+      - name: Install dependencies
```

Comment:
Nitpicking: There is inconsistency in job names capitalization. Vast majority uses Title Case, some of them are Sentence case.

Reply:
I think that there is no Title Case. Only the names of the products/software are capitalized.